src/kuzya.h

```diff
@@ -142,6 +142,7 @@ private slots:
         void slotModificationChanged(bool);
 
         void slotSetFileSuffix(QStringList);
+        void slotSetFileSuffix(QString suffix);
```

Comment:
Signals\slots ideology doesn't let use implicit casting. I thought, there are reason for using overloaded methods